### DISCUSSION TITLE
CSP changes in release - 5.0

### DISCUSSION THREAD

**Discussion by reshmi-nair** [Sunbird-Lern/Community#43] [Contribution]
@chitranshu-keshav, @anunagaraj, @ShailandraOjha

CSP changes in Lern is implemented in release-5.0.0
Please do a regression on the CSP changes in 5.0 Lern release and help with the signoff by Oct 14th.

**Comment by ShailandraOjha** [comment]
Hi, @reshmi-nair as discussed please provide the sign-off from lern env and we will be able to take it up the changes as part of 5.0.2

@sharathkashyap @luckynvdu : Please confirm if you are ok with this.

**Reply by reshmi-nair** [reply]
@ShailandraOjha We have one defect which is blocking the sign-off. So the sign-off is delayed

**Reply by ShailandraOjha** [reply]
Thanx for the update @reshmi-nair as per @luckynvdu We will be adopting these changes in 5.1 as we cannot afford any regression testing in intermediate releases
cc: @chitranshu-keshav @sharathkashyap

**Comment by Ak0094** [comment]
Hello All,

Please find the Sunbird lern Building block 5.0.0 CSP sign-off here.

Environment: [Dev lern](https://dev.lern.sunbird.org/)
Testing Status: Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/issues/?filter=-2&jql=project%20%3D%20LR%20AND%20issuetype%20%3D%20Defect%20AND%20Sprint%20%3D%20243%20order%20by%20created%20DESC)

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1KalQqKtB4KuHgHWRK0B2JINRlyj9H0qJYerDvWwHqNE/edit#gid=0)
**Note**: NA test cases in the execution sheet are related to a new course, certificate which cannot be tested in the dev lern due to environmental limitations. 

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=12667)
Building block sign-off template: [Link](https://docs.google.com/spreadsheets/d/1_Enqc0e62zqEi9wzgwNXKpOsK4hIqLjVvcl8iPf6NSA/edit#gid=0)

cc: @rhwarrier @reshmi-nair @chitranshu-keshav @ShailandraOjha @anunagaraj @Naveenkumar739
